CougSat1-ADCS/src/drivers/DRV8837.h

```diff
@@ -0,0 +1,38 @@
+/******************************************************************************
+ * Copyright (c) 2018 by Cougs in Space - Washington State University         *
+ * Cougs in Space website: cis.vcea.wsu.edu                                   *
+ *                                                                            *
+ * This file is a part of flight and/or ground software for Cougs in Space's  *
+ * satellites. This file is proprietary and confidential.                     *
+ * Unauthorized copying of this file, via any medium is strictly prohibited.  *
+ ******************************************************************************/
+/**
+ * @file DRV8837.h
+ * @author Eric Curtland
+ * @date 27 January 2019
+ * @brief Provides an interface for output to h-bridge 
+*/
+
+#ifndef DRV8837_H_
+#define DRV8837_H_
+
+#include <mbed.h>
+#include "ADCSPins.h"
+
+class DRV8837
+{
+  public:
+    DRV8837(PinName forPin, PinName revPin, PinName pSleep);
+    unit8_t setCoil(float input);
+    unit8_t setCoil(float input, float time);
```

Comment:
Is this not implemented?

Reply:
This just adds the DRV8837 and LTC2499 drivers to the ADCS branch.
I also deleted the unused setCoil function, we decided to implement it's functionality in the event queue system

Reply:
Rename setCoil to setOutput and I'll approve